manager/testdata/e2e/bucket-available.yaml

```diff
@@ -4,9 +4,9 @@
 apiVersion: app.m4d.ibm.com/v1alpha1
 kind: M4DBucket
 metadata:
-  name: test-bucket
+  name: bucket-for-copy
 spec:
-  name: m4d-test-bucket
+  name: datamesh-test-bucket
```

Comment:
why are we using the term "datamesh"? and not "m4d"?

Reply:
outdated artifacts...